foursight_core/react/api/react_api.py

```diff
@@ -1569,10 +1631,71 @@ def reactapi_aws_stack_template(self, request: dict, env: str, stack: str) -> Re
         ignored(request, env)
         return self.create_success_response(aws_get_stack_template(stack))
 
+    def reactapi_accounts_file_upload(self, accounts_file_data: list) -> Response:
+        return self.create_success_response(self._put_accounts_file_data(accounts_file_data))
```

Comment:
Not guaranteed to succeed (but certainly generally should); if it fails should see indication in UI. The _get_accounts_file_data is different and is catching a (certain, no-key) exception because it is okay for the accounts file not to exist, not really an error, just means no account info available (and we indicate that in the UI).